.rubocop.yml

```diff
@@ -55,7 +55,7 @@ Layout/ClassStructure:
       - private_methods
 
 Layout/IndentHeredoc:
-  EnforcedStyle: powerpack
+  EnforcedStyle: squiggly
```

Comment:
Default `EnforcedStyle` of `Layout/IndentHeredoc` cop will be `squiggly`.
https://github.com/rubocop-hq/rubocop/blob/b836478a7b99786677d8377b666211a689f0573c/config/default.yml#L755-L766

So this setting can be removed.

Reply:
The default enforced style for `Style/FrozenStringLiteralComment` will also become `always` after using the latest RuboCop. But can they be config removed already now?

Reply:
It sure seems the can. I’ll squash this change into the 1st commit.